src/features/pages/deriv-ez/how-to-start/data.tsx

```diff
@@ -83,68 +118,151 @@ export const deriv_ez_images = {
     ),
 }
 
-export const demo: React.ComponentProps<typeof StepperView>['items'] = [
+type DemoRealProps = React.ComponentProps<typeof StepperContent>['items']
+const demo: DemoRealProps = [
     {
-        title: '_t_Get the <0>Deriv GO</0> app._t_',
-        component: (
+        title: '_t_<0>Sign in</0> to your Deriv account. If you don’t have one, <1>sign up</1> for free._t_',
+        component: [
             <Link
                 textcolor="brand"
                 key={0}
+                url={{
+                    type: 'company',
+                    url_name: 'deriv_app_login',
```

Comment:
It is not generating any url. You can check the test link

Reply:
good point, thank you! 🙇